### PR TITLE
[HYDRATOR-111] Putting rowkey without any column in Table sink should fail during pipeline publish

### PR DESCRIPTION
**Pull request by shashankmoghe**

### CODE REVIEW

**Comment by shashankmoghe** [comment]
ITN result:

[INFO]
[INFO] integration-test .................................. SUCCESS [6.121s]
[INFO] integration-test-core ............................. SUCCESS [4.231s]
[INFO] integration-test-remote ........................... SUCCESS [16:21.360s]
[INFO] integration-test-standalone ....................... SUCCESS [1.246s]
[INFO] upgrade-test ...................................... SUCCESS [1.918s]
[INFO] long-running-test ................................. SUCCESS [2.079s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16:37.340s
[INFO] Finished at: Wed Dec 14 12:58:28 IST 2016
[INFO] Final Memory: 81M/601M
[INFO] ------------------------------------------------------------------------

File: core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java [hunk 1]
```diff
@@ -62,7 +62,13 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     Schema outputSchema =
       SchemaValidator.validateOutputSchemaAndInputSchemaIfPresent(tableSinkConfig.getSchemaStr(),
                                                                   tableSinkConfig.getRowField(), pipelineConfigurer);
-
+    if ((outputSchema != null) && (outputSchema.getFields().size() == 1)) {
+      String fieldName = outputSchema.getFields().get(0).getName();
+      if (fieldName.equals(tableSinkConfig.getRowField())) {
+        throw new IllegalArgumentException(
+          String.format("Output schema should have columns other than rowkey."));
```
Comment (abhinavbansal246) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java:69]: It would be good to add this in the description file too.

Comment (russorat) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java]: @shashankmoghe can you address this comment?

**Comment by russorat** [comment]
LGTM

**Pull request merged**